sci-libs/hdf5/hdf5-1.10.1.recipe

```diff
@@ -151,5 +151,5 @@ INSTALL()
 
 TEST()
 {
-	make tests
+	make check
```

Comment:
At lines 15 and 16 of the Makefile we have:
```
all lib progs check test _test install uninstall dep depend: _config
	$(MAKE) $@
```
And "check" does not appear anywhere else.

However, at lines 34 and 35 of the Makefile we have:
```
.PHONY: all lib progs test install uninstall dep depend clean mostlyclean     \
	distclean maintainer-clean _config
```

Hence this question: Are you sure it's "make check" that you want, and not "make test" ?

Reply:
"make test" only builds the test cases (and they are already build in the process), it doesn't "actually" do something, "make check" will "actually" run a test :) (checked before)

Reply:
OK. Let's merge, then :)